### DISCUSSION TITLE
What is sql?

### DISCUSSION THREAD

**Discussion by nithin999989** [nithin999989/LeetCodeSolutions#252] [General]
What is sql?

**Comment by git-sumana** [comment]
Response #1
Query Language

**Comment by git-sumana** [comment]
Response #2
SQL (Structured Query Language) is a standardized programming language used for managing and manipulating relational databases.

**Comment by git-sumana** [comment]
> What is sql?

SQL (Structured Query Language) is a standardized programming language used for managing and manipulating relational databases.
